### PR TITLE
Develop rich edit text

### PR DESCRIPTION
**Pull request by githubashutoshsoni**
Bold Italic and strikethrough works when creating a new note.

### CODE REVIEW

**Comment by aritra-tech** [comment]
Hey, @githubashutoshsoni can you please fix the `ktlint` check?
Just run `./gradlew ktlintCheck --continue` in your terminal and check which lines are showing the error and fix it.

**Comment by githubashutoshsoni** [comment]
Sure, it's fixed now. Can you verify?

**Comment by aritra-tech** [comment]
> Sure, it's fixed now. Can you verify?

Yeah sure

File: app/build.gradle.kts [hunk 1]
```diff
@@ -128,6 +128,9 @@ dependencies {
 
     implementation(libs.zoomable)
     implementation(libs.zoomable.image.coil)
+
+    // Rich Edit Text
```
Comment (aritra-tech) [inline app/build.gradle.kts:132]: Can you please write this dependency like others are written please check them out ad write the dependency in version catalog format.

**Review by aritra-tech — CHANGES_REQUESTED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (githubashutoshsoni) [reply app/build.gradle.kts:132]: sure, it's added now.

**Comment by aritra-tech** [comment]
Can you please fix the `ktlint` check

**Comment by aritra-tech** [comment]
WhatsApp Image 2023-10-13 at 13 12 53_11f826fa

Hey, @githubashutoshsoni can you please fix the Notes text field it's looking very bad.
And can you put the `RichTextEditor` at the bottom i.e above the `BottomAppBar`.
I hope you got the points.

**Comment by aritra-tech** [comment]
Hey @githubashutoshsoni any update?
It's almost 1 week, I appreciate your hard work but I also have some rules.

**Comment by aritra-tech** [comment]
Okay, @githubashutoshsoni I am closing this PR.
Thanks for contributing.

**Pull request closed without merging**